CHANGELOG.md

```diff
@@ -14,6 +14,8 @@ Deprecations:
    - `Castle.Core.Internal.Lock` (class)
    - `Castle.Core.Internal.ILockHolder` (interface)
    - `Castle.Core.Internal.IUpgradeableLockHolder` (interface)
+- You should no longer manually emit types into DynamicProxy's dynamic assembly. For this reason, the following member has been deprecated. (@stakx, #445)
+   - `Castle.DynamicProxy.ModuleScope.DefineTye` (method)
```

Comment:
Spelling of method name

Reply:
Oh, gods! I should start to spell-check my changelog entries. :facepalm: